mon-pix/translations/en.json

```
@@ -525,6 +525,9 @@
                 "subtitle": "Discover the skills recommended for you.",
                 "title": "Recommended skills"
             },
+            "score": {
+                "profile-link": ""
```

```suggestion
                "profile-link": "See my skills"
```
On a traduit come ça sure les autres liens.